pcsx2/PINE.cpp

```diff
@@ -623,7 +624,7 @@ PINEServer::IPCBuffer PINEServer::ParseCommand(std::span<u8> buf, std::vector<u8
 					goto error;
 				if (!SafetyChecks(buf_cnt, 1, ret_cnt, 0, buf_size)) [[unlikely]]
 					goto error;
-				VMManager::SaveStateToSlot(FromSpan<u8>(buf, buf_cnt));
+				Host::RunOnCPUThread([buf, buf_cnt] { VMManager::SaveStateToSlot(FromSpan<u8>(buf, buf_cnt)); });
```

Comment:
Buffer isn't guaranteed to be the same or alive on the CPU thread.
```suggestion
				Host::RunOnCPUThread([slot = FromSpan<u8>(buf, buf_cnt)] { VMManager::SaveStateToSlot(slot); });
```

Ditto for the other one below.

Reply:
ack, thanks!